packages/sidecar/package.json

```diff
@@ -38,8 +38,16 @@
       "types": "./dist/parser/index.d.ts"
     },
     "./format": {
-      "import": "./dist/format/index.js",
-      "types": "./dist/format/index.d.ts"
+      "import": "./dist/formatters/index.js",
+      "types": "./dist/formatters/index.d.ts"
+    },
+    "./formatters/human/errors": {
```

Comment:
Not a fan of having to export the submodules explicitly like this for every formatter. Maybe we should keep them in a single file pee formatter?